src/binding/python/Container.cpp

```diff
@@ -141,6 +143,16 @@ namespace detail
             >()
         );
 
+        cl.def_property_readonly(
```

Comment:
Is this your inspiration for #255?

Reply:
yep, can expose it in python easily as property behind the scenes, but in C++ #255 we could make this already pretty